backend/multi/backend.c

```diff
@@ -62,10 +62,24 @@ static struct wlr_egl *multi_backend_get_egl(struct wlr_backend *wlr_backend) {
 	return NULL;
 }
 
+static struct wlr_renderer *multi_backend_get_renderer(
+		struct wlr_backend *backend) {
```

Comment:
It's a separate issue, but I believe this operation should not be supported for the multi-backend because it could have multiple renderers.

Reply:
We do need it for the multi libinput+drm, just like EGL.